solutions-ui-navbar/webpack.config.prd.js

```diff
@@ -1,11 +1,12 @@
 const path = require("path");
+const {name, version} = require("./package.json");
 module.exports = {
   mode: "production",
   devtool: false,
   entry: "./src/index.js",
   output: {
     path: path.resolve(__dirname, "dist"),
-    filename: "[name].js",
+    filename: `${name}.${version}.js`,
```

Comment:
I believe we do want to benefit from browser caching to fetch it. If we want to deliver a new version we can increment the version accordingly.

Reply:
Ah, got it, it's because we want to have a deterministic name for easier inclusion in other UIs through `script` tags. OK, fine with me, I guess it's not a problem if this navbar version changes a lot more than MetalK8s itself :)